content/sponsors/receiving-sponsorships-through-github-sponsors/unpublishing-your-github-sponsors-profile.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Unpublishing your GitHub Sponsors profile
-intro: 'If you don''t have any sponsors, you can unpublish your {% data variables.product.prodname_sponsors %} profile.'
+intro: "If you don't have any sponsors, you can unpublish your {% data variables.product.prodname_sponsors %} profile."
```

Comment:
Hey @Panquesito7! Thanks for opening up this follow-up PR. 

Heads up that the extra single quote here works as an escape character and our standard is to use single quotes in the intro.

Here's a preview of this article on staging:  
![image](https://user-images.githubusercontent.com/6732600/117705650-27b4f400-b181-11eb-8c10-c75aa3254211.png)

You can check your PR's changes on staging by going to the CI tests, scrolling down to "Staging" and clicking "Details".

The Details link should direct you to a unique URL that shows the changes in your PR. 

![image](https://user-images.githubusercontent.com/6732600/117708859-2a194d00-b185-11eb-90a9-cb38c238fd63.png)

I wonder if something else you could add to these Sponsors articles in this PR or another PR. 🤔

Reply:
Hmm, I think I will close this PR then.
Thank you for your help anyways! 😄